### DISCUSSION TITLE
[M-Z] (9 Settembre 2019) Esercizio 3

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#449] [Esami & Esoneri M-Z]
### Traccia

<img width="581" alt="image" src="https://github.com/user-attachments/assets/d65c6b0d-aa9d-40e7-84a4-f06134d0a254">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190909.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190909.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/ef4850db-d430-41b0-b9e8-e4fa5092ed02)
